src/main/asciidoc/index.adoc

```diff
@@ -33,6 +33,7 @@ The library consists of these modules:
 *belgif-rest-problem:*
 
 * Removed deprecated InvalidParamProblem: use InputValidationProblem, which supports both the legacy invalidParams[] and the new issues[] structure
+* Added requireIfPresent check for validating input(s) that must be present when a given target input is present
```

Comment:
Probably best to add it to a *belgif-rest-problem-validator* section as that's the module with the change